bin/console

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env php
+<?php
```

Comment:
Not sure we should even include this file. You won't be able to add new commands unless you implement it yourself.